docs/Information_nodes.rst

```diff
@@ -70,6 +70,60 @@ commands::
 
     --gres=gpu:a100:<number> --reservation=DGXA100
 
+MIG
+^^^
+
+.. _mig_nodes:
+
+MIG (`Multi-Instance GPU <https://www.nvidia.com/en-us/technologies/multi-instance-gpu/>`_)
+is an NVIDIA technology allowing certain GPUs to be
+partitioned into multiple *instances*, each of which has a roughly proportional
+amount of compute resources, device memory and bandwidth to that memory.
+
+NVIDIA supports MIG on its A100 GPUs and allows slicing the A100 into up to 7
```

Comment:
theoretically allows ?

Reply:
I think it's ok as it is but I'll let @obilaniu decide. However, if tomorrow same time this is not merged I'll press the button ;)

Reply:
The "theoretically" discussion serves to introduce the explanation that the static-partitioning issue isn't NVIDIA's but SLURM's fault, and thus that this partitioning isn't set in stone and is changeable (and we likely will change it). I'd like to keep it but if it looks too much like going into the weeds, we can take it out.